app/src/main/java/com/antailbaxt3r/collegemate/activities/LoginActivity.java

```diff
@@ -146,18 +154,32 @@ public void onComplete(@NonNull Task<AuthResult> task) {
                 });
     }
 
+    private void oAuthVerification(String idtoken){
+        APIInterface apiInterface = RetrofitClient.getClient();
+        apiInterface.sendRegToken(idtoken).enqueue(new Callback<String>() {
+            @Override
+            public void onResponse(Call<String> call, Response<String> response) {
+                Log.d("oAuth","Successfull");
+                firebaseAuthWithGoogle(idtoken);
+            }
+
+            @Override
+            public void onFailure(Call<String> call, Throwable t) {
+                Log.d("error",t.getMessage());
+            }
+        });
+    }
+
     private void gotoHome() {
         Intent intent = new Intent(LoginActivity.this, MainActivity.class);
         intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
         startActivity(intent);
-        finish();
     }
 
-    private void goToOnboarding() {
-        Intent intent = new Intent(LoginActivity.this, MainActivity.class);
-        intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
-        startActivity(intent);
-        finish();
```

Comment:
Don't remove this. This is fine

Reply:
Two Functions with same functionality was present. Seemed unnecessary.